webtau-feature-testing/test-expectations/scenarios/concept/conditionalEnvRegistrationSkip/run-details.json

```diff
@@ -3,6 +3,12 @@
     "scenario" : "this scenario will only be executed in \"experimental\" env",
     "shortContainerId" : "conditionalEnvRegistrationSkip.groovy",
     "stepsSummary" : { }
+  }, {
+    "scenario" : "this scenario will not be executed in \"experimental\" env",
+    "shortContainerId" : "conditionalEnvRegistrationSkip.groovy",
+    "stepsSummary" : {
```

Comment:
I may have forgotten how skip works. Does it make scenario as failed?

Reply:
it's like a `dscenario`

Reply:
right, so that would give exit code `1`, what makes it a failed step inside?

Reply:
ah never mind, re-read the test